pkg/virt-operator/resource/generate/components/deployments.go

```diff
@@ -40,7 +40,11 @@ const (
 
 	VirtAPIName        = "virt-api"
 	VirtControllerName = "virt-controller"
-	VirtOperatorName   = "virt-operator"
+	VirtOperatorName   = "virt-operator"	
+	PrometheusUrl	   = "prometheus.kubevirt.io"
```

Comment:
That these look like urls is not the important part in this case. Consider naming this `PrometheusAppLabelServiceLabel` or something.